src/structures/Guild.js

```diff
@@ -716,7 +717,7 @@ class Guild extends AnonymousGuild {
    *   .catch(console.error);
    */
   async fetchVanityData() {
-    if (!this.features.includes('VANITY_URL')) {
+    if (!this.features.includes('VANITY_URL') && !this.features.includes('GUILD_WEB_PAGE_VANITY_URL')) {
```

Comment:
Here you're checking if they have both the `VANITY_URL` _and_ the `GUILD_WEB_PAGE_VANITY_URL` feature. But the error states that the guild does not have the `VANITY_URL` _or_ the `GUILD_WEB_PAGE_VANITY_URL` feature.

Reply:
You are wrong, i am checking if it doesnt have all of these

Reply:
Oops my bad I didn't see the `!`, you can ignore this suggested change